packages/_example/src/forest/customizations/dvd.ts

```diff
@@ -28,6 +28,10 @@ export default (collection: DvdCustomizer) =>
           ids: await context.getRecordIds(),
         };
 
+        // Price increase above 50% needs to be validated by a manager !
+        // This feature needs a Paying Plan.
+        if (context.formValues.percentage > 50) return resultBuilder.requestApproval();
```

Comment:
```suggestion
        if (context.formValues.percentage > 50) resultBuilder.requestApproval();
```
using `throw` method